### PR TITLE
Fourth Quadrant Fix

### PR DESCRIPTION
**Pull request by m4rtint**
- Fixed the debug mode on game
- removed Main Camera Tag from Doozy Main Menu since the Main Mneu camera is the one being changed, instead of the Map Camera
- Added Tests for Utilitiy

### CODE REVIEW

**Comment by m4rtint** [comment]
[Test Utilities](https://trello.com/c/SjUPAFD0/538-test-utilities)

**Comment by m4rtint** [comment]
[Fourth screen on three-player game freezes in corner map](https://trello.com/c/hFhxR16S/626-fourth-screen-on-three-player-game-freezes-in-corner-map)

File: Assets/_PartyParkade/Tests/Utilities_Tests.cs [hunk 1]
```diff
@@ -65,5 +65,171 @@ public void ColorFrom255_FullWhite_EqualWhite()
             //Assert
             Assert.AreEqual(actualValue, Color.white, "Values should be equally white");
         }
+
+        
+        /*
+         *                                               x,  y, w,   h
+            * this.m_twoPlayers.Add(new ScreenAttributes(0f, 0f, 0.5, 1));
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Tests/Utilities_Tests.cs:72]: any reason why this is commented out?

Comment (m4rtint) [reply Assets/_PartyParkade/Tests/Utilities_Tests.cs:72]: This is just for reference as to what the specific values are supposed to be.

**Pull request merged**